deploy.env

```diff
@@ -1,23 +1,39 @@
 # This file is a template for docker compose deployment
 # Copy this file to .env and change the values as needed
 
-# PostgreSQL default user password
-POSTGRES_PASSWORD="changepassword"
+# PostgreSQL Settings
+POSTGRES_HOST=postgres
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=changepassword
+POSTGRES_PORT=5432
+POSTGRES_DB=postgres
+
+# Supabase user settings
+SUPABASE_USER=supabase_auth_admin
+SUPABASE_PASSWORD=root
+
+# Redis Settings
+REDIS_HOST=redis
+REDIS_PORT=6379
+
+# Gotrue Settings
+GOTRUE_HOST=gotrue
+GOTRUE_PORT=9999
 
 # AppFlowy Cloud
 ## URL that connects to the gotrue docker container
-APPFLOWY_GOTRUE_BASE_URL=http://gotrue:9999
+APPFLOWY_GOTRUE_BASE_URL=http://${GOTRUE_HOST}:${GOTRUE_PORT}
```

Comment:
There's actually no reason to change gotrue host, since we are not going to use the publicly accessible endpoint. We can keep it as gotrue:9999.

This applies to Redis as well.